client/ayon_core/modules/deadline/plugins/publish/submit_houdini_render_deadline.py

```diff
@@ -265,20 +266,23 @@ def get_plugin_info(self, job_type=None):
         # Output driver to render
         if job_type == "render":
             product_type = instance.data.get("productType")
-            if product_type == "arnold_rop":
+            rop_node = hou.node(instance.data.get("instance_node"))
+            node_type = rop_node.type().name()
+
+            if node_type == "arnold":
                 plugin_info = ArnoldRenderDeadlinePluginInfo(
                     InputFile=instance.data["ifdFile"]
                 )
-            elif product_type == "mantra_rop":
+            elif node_type == "ifd":
                 plugin_info = MantraRenderDeadlinePluginInfo(
                     SceneFile=instance.data["ifdFile"],
                     Version=hou_major_minor,
                 )
-            elif product_type == "vray_rop":
+            elif node_type == "vray_renderer":
                 plugin_info = VrayRenderPluginInfo(
                     InputFilename=instance.data["ifdFile"],
                 )
-            elif product_type == "redshift_rop":
+            elif node_type == "Redshift_ROP":
```

Comment:
What's the reason we prefer the node type over the product type now?

Reply:
Nothing. 
I should have reverted it after removing the intermediate family. 
https://github.com/ynput/ayon-core/pull/328/commits/4e6bd3d3361708fa7cf84ef69e0e5715ed451df2

Reply:
done in https://github.com/ynput/ayon-core/pull/328/commits/75879f54be31e293cad18a1c4a8a60005d62ba70